modules/Hoot/tools/folderTree.js

```diff
@@ -436,7 +436,12 @@ export default class FolderTree extends EventEmitter {
         let { data } = d;
 
         if ( data.type === 'folder' ) {
-            return (data.public) ? '#7092ff' : '#efefef';
+            if ( data.selected ) {
+                return '#ffff99';
+            }
+            else {
```

Comment:
don't really need the else here since the if has a `return`.

Reply:
removed else